src/bin/common/solving_loop.ml

```diff
@@ -471,6 +480,7 @@ let main () =
     State.empty
     |> State.set solver_ctx_key solver_ctx
     |> State.set partial_model_key partial_model
+    |> set_steps_bound (O.get_steps_bound ())
```

Comment:
Should this be taken care of by `DO.init`?

Reply:
We don't even need to initialize this variable